.gitignore

```diff
@@ -1,9 +1,8 @@
 *.iso
 *.raw
+venv
 base_image
-include.d/disk-layout.yml
-include.d/runcmd.yml
-user-data
-user-data.yml
 *.iso
 *.iso.CHECKSUM
+.idea
```

Comment:
Ignores that refer to a specific development environment should be done in the global git config of the developer